ups/product_deps

```diff
@@ -7,7 +7,7 @@ fcldir product_dir job
 fwdir  product_dir config_data
 #
 product         version
-larsim		v08_33_00
+larsim		v08_32_00
```

Comment:
Changes to `ups/product_deps` should not be included in the PR; they will be included as part of the LArSoft release.